InstagramStories/Source/CollectionViewCell/IGStoryPreviewCell.swift

```diff
@@ -58,13 +62,17 @@ final class IGStoryPreviewCell: UICollectionViewCell, UIScrollViewDelegate {
         return (snapIndex == 0) ? 0 : scrollview.subviews[previousSnapIndex].frame.maxX
     }
     private var videoSnapIndex: Int = 0
-    //private var videoView: IGPlayerView?
     
     var retryBtn: IGRetryLoaderButton!
     
     //MARK:- Public iVars
     public var direction: SnapMovementDirectionState = .forward
-
+    public var getScrollView: UIScrollView {
```

Comment:
What is the purpose of this variable? and do we really need this? because I hope we have already kept the scrollview instance inside the IGStoryPreview Cell

Reply:
That scrollView scope is private. If we don't need this variable then we need to make that as Public. Is that fine?

Reply:
change it to public

Reply:
okay